src/components/AgenciesPage/components/components/AgenciesItem.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Card } from '@mui/material';
+import { AgenciesItemImage } from './components/AgenciesItemImage';
+import { AgenciesItemContent } from './components/AgenciesItemContent';
+import { AgenciesItemActions } from './components/AgenciesItemActions';
+import './AgenciesItem.scss';
+import type { AgenciesItemProps } from '../../types/Agencies';
+
+export const AgenciesItem: React.FC<AgenciesItemProps> = ({ agency }) => (
```

Comment:
Should we typing components as React.FC or it isnt necessary ?

Reply:
we cat use another approach: ({ agency : AgenciesItemProps})
But in this case, our type should  contain children's prop if we are going to use it 
that's why I prefer my option and consider it as more readable

Reply:
You should leave your option and also add Dmytro's :)

Reply:
do you mean this? 

`export const AgenciesItem: React.FC<AgenciesItemProps> = ({ agency }: AgenciesItemProps)`

Reply:
fixed in this way